doc/sphinx/using/libraries/funind.rst

```diff
@@ -169,14 +169,21 @@ terminating functions.
 Tactics
 -------
 
-.. tacn:: functional induction (@qualid {+ @term})
+.. tacn:: functional induction @term {? using @one_term {? with @bindings } } {? as @simple_intropattern }
```

Comment:
The note below on `(@qualid ...)` makes no sense anymore after this change. It should be modified to say `@term` should be `@qualid ...` where `@qualid`... The second note about non-mandatory parentheses should be dropped.

Reply:
Not sure I understood you.  Check what I did.

Reply:
Maybe just remove the `...`
But thinking further, I'm wondering if we shouldn't remove the note and instead integrate its content (minus the note on implicit arguments which is *not* interesting) to the main description. I.e., something like:

>Performs case analysis and induction following the definition of a function `@qualid` (which must be fully applied to its arguments as part of `@term`).

Reply:
OK, dropped the note.  Does "the definition of a function" clearly imply something defined with `Function`?  If not, what should I add?

Reply:
I think it is meant implicitly (although I'm not sure) but it's certainly not going to be clear to all readers.